fastlane/Fastfile

```diff
@@ -150,6 +150,17 @@ platform :ios do
       UI.error '`oclint` found a problem. Please check ' + ocpath
     end
 
+    if Dir.glob("../*.podspec").length > 0
+      begin
+        pod_lib_lint(allow_warnings: true)
```

Comment:
If I run this on push sdk it fails because it can't find a dependency's podspec - the internal spec repo needs to be passed in as a source. I think you can call the `prd_lint` lane which uses the private `lint_podspec` lane.